libnavigator/src/test/java/com/mapbox/navigation/navigator/NavigationOkHttpServiceTest.kt

```diff
@@ -36,13 +36,15 @@ class NavigationOkHttpServiceTest {
     private val httpClientBuilder: OkHttpClient.Builder = mockk(relaxUnitFun = true)
     private val httpClient: OkHttpClient = mockk(relaxUnitFun = true)
     private val httpDispatcher: Dispatcher = mockk(relaxUnitFun = true)
+    private val body: ByteArray = byteArrayOf()
 
     private val nativeRequest = HttpRequest(
         HttpMethod.GET,
         "https://api.mapbox.com/xyz",
         hashMapOf("User-Agent" to "MapboxNavigationNative/X.Y.Z MapboxNavigationNative"),
         true,
-        UserAgentComponents.Builder().build()
+        UserAgentComponents.Builder().build(),
+        body
```

Comment:
@korshaknn could you check that this change is correct? `mapboxCommon` `6.1.0` introduced some changes and not sure if fixing the test is enough or if we might need to do changes in `NavigationOkHttpService`.

Reply:
cc @LukasPaczos

Reply:
Yes, I think we're good.